src/App.js

```diff
@@ -0,0 +1,55 @@
+import { Navbar, Nav, NavItem, Button } from 'react-bootstrap';
+import React, { Component } from 'react';
+import Home from './Home';
+import PathFinder from './PathFinder';
+import FundsTransfer from './FundsTransfer'
+import './App.css';
+
+class App extends Component {
+constructor() {
+  super()
+  this.handleNav = this.handleNav.bind(this)
```

Comment:
you don't need this line because it's already a `handleNav = (e) => {` (field class property, are already bound to the instance)